### PR TITLE
Copy over Admin UI base for ctl-admin-ui

### PR DESCRIPTION
**Pull request by elliotbonneville**
### Code Changes

* [x] Create a `clients` directory
* [x] Create an `admin-ui` directory containing copied over directory from `admin-ui`
* [x] Remove Admin UI specific features from `admin-ui` and clean up

### Steps to Confirm

* [ ] _list any manual steps taken to confirm the changes_

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation Updated:
  - [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`

### CODE REVIEW

**Review by allisonking — APPROVED** [review]
woot, lgtm! would just want to double check with @ThomasLaPiana that this code lives in the right folder (`src/clients/ctl-admin-ui`)

File: src/clients/ctl-admin-ui/__tests__/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// __tests__/index.test.tsx
+import { render, screen } from '@testing-library/react';
```
[the remark this answers is not in the document]
Comment (ThomasLaPiana) [reply src/clients/ctl-admin-ui/__tests__/index.test.tsx:2]: > woot, lgtm! would just want to double check with @ThomasLaPiana that this code lives in the right folder (`src/clients/ctl-admin-ui`)

I don't have any strong opinions here as long as everything is logically organized 😄

**Comment by ThomasLaPiana** [comment]
> woot, lgtm! would just want to double check with @ThomasLaPiana that this code lives in the right folder (`src/clients/ctl-admin-ui`)

I don't have any strong opinions here as long as everything is logically organized 😄  @allisonking

**Comment by ThomasLaPiana** [comment]
> woot, lgtm! would just want to double check with @ThomasLaPiana that this code lives in the right folder (`src/clients/ctl-admin-ui`)

I don't have any strong opinions here as long as everything is logically organized 😄  @allisonking

**Comment by NevilleS** [comment]
This is super nitpicky, but can we use `admin-ui` instead of `ctl-admin-ui`?

It's just a bit inconsistent across projects to call it `admin-ui` on fidesops, but `ctl-admin-ui` on fidesctl, you know?

File: src/clients/ctl-admin-ui/.env.development [hunk 2]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_FIDESOPS_API=http://0.0.0.0:8080/api/v1
```
Comment (ThomasLaPiana) [inline src/clients/ctl-admin-ui/.env.development:1]:
```suggestion
NEXT_PUBLIC_FIDESCTL_API=http://0.0.0.0:8080/api/v1
```

**Comment by allisonking** [comment]
Does anyone know where I can get an svg of the fidesctl logo to replace the fidesops one?

File: src/clients/ctl-admin-ui/src/features/common/Icon/ArrowDownLine.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { createIcon } from '@fidesui/react';
+
+export default createIcon({
+  displayName: 'ArrowDownLineIcon',
+  viewBox: '0 0 24 24',
+  d: 'M12 13.1719L16.95 8.22192L18.364 9.63592L12 15.9999L5.63599 9.63592L7.04999 8.22192L12 13.1719Z',
```
[the remark this answers is not in the document]
Comment (elliotbonneville) [reply src/clients/ctl-admin-ui/src/features/common/Icon/ArrowDownLine.tsx]: I navigated to the appropriate SVG element in Figma, then right clicked it and clicked "Copy as SVG".

**Comment by elliotbonneville** [comment]
@allisonking See my other recent comment.

@NevilleS Agreed re. name. Also, for further nitpicky consistency I noticed that `admin-ui` lives in `fidesops/clients` but this directory would be `fides/src/clients`. We should either move `fidesops/clients` to `fidesops/src/clients` or `fides/src/clients/ctx-admin-ui` to `fides/clients/admin-ui`. Thoughts, all?

File: src/clients/ctl-admin-ui/src/features/common/Icon/ArrowDownLine.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (elliotbonneville) [reply src/clients/ctl-admin-ui/src/features/common/Icon/ArrowDownLine.tsx]: (That gave me the body of the SVG, which I then copied into the `d` param of the Chakra icon creation options object).

[the remark this answers is not in the document]
Comment (allisonking) [reply src/clients/ctl-admin-ui/src/features/common/Icon/ArrowDownLine.tsx]: beautiful, thank you!

**Review by ThomasLaPiana — CHANGES_REQUESTED** [review]
`internal` shouldn't get committed in the `fidesctl.toml`

**Comment by allisonking** [comment]
> @elliotbonneville easier for us to match fidesops than change fidesops I think. Let's go ahead and match and then ship it!

okay, I can change and clean up the toml at the same time

**Review by ThomasLaPiana — APPROVED** [review]
Ship it!

**Pull request merged**